pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/form/block/question/QuestionCreatorHelper.java

```diff
@@ -246,7 +251,22 @@ PicklistQuestion createPicklistQuestion(AIBuilderContext ctx, PicklistQuestionDe
                                     .createPicklistOption(ctx, picklistOptionDef, questionDef.getGroups())));
         }
 
-        List<PicklistOption> picklistOptions = CollectionMiscUtil.createListFromAnotherList(questionDef.getLocalPicklistOptions(),
+        //localPicklistOptions doesn't include RemotePicklistOptions
+        //include option defs of any already selected options (answers)
+        List<PicklistAnswer> answers = questionCreator.getAnswers(ctx, questionDef.getStableId());
+        List<PicklistOptionDef> options = questionDef.getLocalPicklistOptions();
+        if (questionDef.getRenderMode() == PicklistRenderMode.REMOTE_AUTOCOMPLETE && !answers.isEmpty()) {
+
+            List<String> selectedOptsStableIds = answers.stream()
+                    .flatMap(answer -> answer.getValue().stream().map(SelectedPicklistOption::getStableId))
+                    .collect(Collectors.toList());
```

Comment:
Isn't it safer to rewrite it this way:
`
List<String> selectedOptsStableIds = answers.stream()
                    .map(Answer::getValue)
                    .flatMap(ValueClass::steam)
                    .map(SelectedPicklistOption::getStableId))
                    .collect(Collectors.toList());
`
In this case we could probably avoid NPE in case answer's value is null for instance.